### DISCUSSION TITLE
Mobs dont spawn in mob farms

### DISCUSSION THREAD

**Discussion by GoncaloT** [GriefPrevention/GriefPrevention#1249] [Q&A]
Hello! I think that i have some plugin that have an config that the mobs dont spawn in claims our wherever i just dont know why in my server dont spawn mobs in my mob farms and i´m doing the mob farms correctly

**Comment by Jikoo** [comment]
Impossible to help you without more info - that is not a default GP feature. GP only ever messes with spawning in creative worlds.
